### ISSUE TITLE
Most guns literally smells skill issue

### ISSUE THREAD

**Issue by snipergamingop** [deferred-impact/battledudes-issue-tracker#71]
Yeah that's right, most AR can be safely put simple like
AK: omg i need more dmg so i can kill people faster with fewer bullet
M4: all the other AR sucks for me so imma use this
SCAR: imma ambush people
Falcon: haha high dmg go ding ding
FAMAS: people say that bulletspam is bad so imma use this (still bulletspamming OMEGALUL)
M249: i need more bullets to fire since im so bad at aiming and i can W+M1 now
And that's just AR alone, there's SMGs and shotguns as well

**Comment by snipergamingop** [comment]
My suggestions? Lower the accuracy + firerate of all the automatic weapon, maybe lower the reserve ammo as well.

**Comment by snipergamingop** [comment]
Lower like 25-35% of reserve ammo of all amutomatic weapons to limit them, or even lower the resupply rate of ammunition in general

**Comment by Gregory-Y** [comment]
Indeed. As deferred impact put it, bd = 0 skill bullet spam fest. I would suggest implementing the same fix as the one that was added to the M249, make all weapons become a lot less accurate after 4-5 shots are fired in quick succession. Lowering the reserve ammo by 25% is also something I support. I would go even further to decrease the damage, and I'm thinking about creating an updated version of the weapon balance compendium issue to remedy that.

The only problem is how would mobile players then be able to play the game because using automatic weapons would then become much harder. Mobile gamer's aren't exactly at an advantage when it comes to being able to easily exercise fire discipline. Its pretty much drag and shoot for them and most of them are casuals. They can't tapfire or burstfire the same way PC players can and most of them probably aren't willing to do that. I'd imagine they are just playing bd to pass the time and I don't think they would particularly enjoy gameplay that requires them to properly time their bursts. Though, maybe that's for the better anyway.

While I think nerfing automatic weapons like rifles and smgs would be great I also think it could filter out a good portion of the playerbase and cause bd to further lose popularity, so I just want to point that out so that whatever nerfs you have in mind don't change the gameplay into snipers only or something like that.

**Comment by deferred-impact** [comment]
TLDR: improve the accuracy. Decrease the damage.

This shouldn't be surprising for you to hear; but then at the same time you keep suggesting inaccuracy, so perhaps it will be surprising to hear: **i dislike inaccuracy**. It's a terrible concept in any circumstances, but especially if you are trying to argue that bulletspam is low skill.

Yes, bulletspam is low skill. Obviously so. The risk of missing a single shot is minimal, and these weapons are balanced with "the user will miss 50%+ of their shots" in mind.

**But,** inaccuracy is not the solution. Why? The answer is simple. **Because it lowers the skill even further.** Inaccuracy is luck, and luck pretty much always displaces skill. I do not want to play a game in which the outcome of a battle depends largely on where my bullets *decided on their own* to go.

Today, you get the dev to nerf the accuracy. Tomorrow, the bulletspammers start loudly crying and get the dev to buff the damage. The weapons are now back to being just as powerful, so the day after, you convince the dev to lower the accuracy even more. On and on, and on... can you see where we are going?

Of course, personally, i'd nerf all bulletspam guns into the goddamn ground. But, y'know. **Good luck. With that.** Closest we ever got was perhaps season 7, where ARs were noticeably less powerful (yet still *the* weapon of choice for high KPG and just in general). All it took is one season of (not even particularly loud) whining from bulletspammers, and we're back to assault rifle supremacy.

Lacking the option to nerf these guns into the asphalt, (or delete them outright - **and i definitely would if i could**), the only alternative i see is to push them in the opposite direction. **More** accuracy, **less** damage. Less room for error. If we cannot delete them, perhaps we could at least turn them into quasi-snipers. Yes, missing still mostly doesn't matter, but at least this way every miss is undeniably the fault of the user, and so we can at least begin speaking of this "skill" thingy that seems to matter.

The other benefit this would bring, is that it would still allow for inaccuracy if need be. Nothing stops you from just wildly swinging your mouse around if you absolutely must. However, with the right balancing, it would be a rather dumb idea to do so.

**Comment by deferred-impact** [comment]
> The only problem is how would mobile players then be able to play the game because using automatic weapons would then become much harder. Mobile gamer's aren't exactly at an advantage when it comes to being able to easily exercise fire discipline.

Addressed by #60

**Comment by Gregory-Y** [comment]
Well the season 7 rifles were rather poorly balanced too. Like if you're gonna decrease the damage of the ak, falcon, and scar then decrease the m4 and m249's damage too, that just resulted in the M4 becoming the premier spamming weapon while the other automatic are made completely irrelevant. But I get your point kinda. I'll modify my suggestions in the weapon balance compendium.

**Comment by deferred-impact** [comment]
True, there was imbalance between ARs. But that's not my point. You should consider how it was fixed. Instead of nerfing the strong rifles, the dev decided to buff the weak rifles.

I highly doubt we'll ever get the ARs nerfed to the point where HS becomes *the* gun to use, because tons of players will always veto that move. So instead i propose we rebalance rifles around being more accurate, but less damaging. Their effectiveness doesn't need to change, but they will take more skill to use

**Comment by snipergamingop** [comment]
And that's just ARs , there are SMGs and Shotguns as well. The SMGs can be rebalanced in a similar fashion, but instead of nerfing the damage, I suggest nerfing the falloff a bit so it would be a close quarters DPS weapon. The shotgun, however is not that simple.
